src/components/network/network-panel.js

```diff
@@ -135,8 +148,21 @@ genotet.NetworkPanel.prototype.setEdgeInfo_ = function(edge, container) {
     .text(edge.source.label);
   container.children('#target').children('span')
     .text(edge.target.label);
-  container.children('#weight').children('span')
+  container.children('#edge-sub-info').children('#weight').children('span')
     .text(edge.weight);
+  container.children('#edge-sub-info').children('#rm-edge').children('button')
+    .click(function() {
+    this.signal('update', {
+      type: 'delete-edges',
+      edges: [{
```

Comment:
If you make sure you pass in genotet.networkEdge then you can just use [edge] here.